styles.css

```diff
@@ -221,25 +222,30 @@ li {
   padding: 12px;
 }
 
-/* Work section*/
+/* Work section */
 
 .Works {
-  display: grid;
+  display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 120px 0px;
+  padding: 120px 0.001px;
   background: #fdfdfd;
   position: absolute;
   width: 100%;
   height: 3658.4px;
-  left: 0px;top: 682px;
-  font-family: Crete Round;
+  left: 0.001px;
+  top: 682px;
+
+  /* decoration */
+  font-family: Crete Round, sans-serif;
   font-style: normal;
   font-weight: normal;
-  font-size: 40px;
+  font-size: 32px;
   line-height: 52px;
-  display: flex;
-  align-items: center;
+  color: #1c1a19;
+
+  /* background-image: url(./images/works/img-works-0.svg);
```

Comment:
[OPTIONAL]
**Keep good practices and professionalism.**
- Remove commented code.